packages/react-ui/src/components/molecules/Alert.js

```diff
@@ -0,0 +1,117 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { Alert as MuiAlert, AlertTitle, Fade, styled } from '@mui/material';
+import Typography from '../atoms/Typography';
+
+const StyledAlert = styled(MuiAlert, {
```

Comment:
Please move the styles to the theme, in feedback section: packages/react-ui/src/theme/sections/components/feedback.js

Otherwise the Mui Alerts are going to keep the old design.

You have here an example about how to style custom props:  packages/react-ui/src/theme/sections/components/buttons.js#L166

Reply:
@vmilan I have done it here https://github.com/CartoDB/carto-react/pull/698/commits/60316ece138e5abc772e9df1034f712c26bbdd24
Is this the right way? There is something I'm not sure, it's the custom props I need to pass to MuiAlert… I haven't found the similar example you pointed to me. And I don't understand how it can work with Typescript and without `styled`. So there is probably something wrong here.

Reply:
Ooh, and even if I hate it I had to add an `!important` [here](https://github.com/CartoDB/carto-react/pull/698/files#diff-da0b30fb3a340c474f2342cf091a8130a7fde3142076a5a0315d750a98b40961R71), because otherwise the color wasn't working well when put in theme… The CSS scopes are different than with `styled`, I hope there are not other bad surprise

Reply:
The recommended way of styling custom variants and props is documented here: https://mui.com/material-ui/customization/theme-components/#creating-new-component-variants

But if you think is not going to work (probably right), the best way IMO is to follow the same approach as the `Avatar`.

Tthe common styles are in the theme, and the new props are styled in the component: 
- packages/react-ui/src/theme/sections/components/dataDisplay.js#L475
- packages/react-ui/src/components/molecules/Avatar.js

Since these are new features of our Alert component (`neutral` severity and `content` prop), I don't see any problem on keep them in a styled component(s) in the Alert component itself.

Reply:
Ok, I ended up letting most of it in the component itself, because it's mostly layout changes depending on new props that wouldn't work well for MUI Alerts, but I could keep some light stuff like border radius and title styles in the theme override.
https://github.com/CartoDB/carto-react/pull/698/commits/7ab73ca3e6ee666c18cf4e7f4495e11cfb5a7a06